pkg/openstack/placement.go

```diff
@@ -130,6 +131,7 @@ func ReconcilePlacementAPI(ctx context.Context, instance *corev1beta1.OpenStackC
 	}
 
 	if placementAPI.IsReady() {
```

Comment:
observedGeneration is available since https://github.com/openstack-k8s-operators/placement-operator/commit/c710647a7745767a1004728d99d9eb62759ba43f

Reply:
I don't think we should chase these now. We can do them in a follow up

Reply:
OK